react-ui/src/pages/entry/Entry.jsx

```diff
@@ -1,30 +1,28 @@
 /**
  * Full page entry
+ * ??? redirect if !entry
```

Comment:
Part of the planned #12 work is 404 pages (which you've added, so thanks!) and handling errors like missing IDs. In this case, I'd prefer showing a message that entry is not found, click to view contest, or if contest isn't found, click to view contests, but that's a future task.